### ISSUE TITLE
Username must not be required

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-backend#654]
One of the fields [`first_name`, `last_name`, `username`] must be required.
If a `username` is not provided, a username must be generated from first_name and/or last_name by the backend.

**Comment by jsangmeister** [comment]
See #647: AFAIK, you wanted to move this generation to the client and the server expects the username to be set. Is that correct @FinnStutzenstein?

**Comment by FinnStutzenstein** [comment]
No, `username` is required. The client must calculate the username from title/first_name/last_name: join all non-empty with a space.

For the client: This can be entered in the form when the user clicks `ok`. If the generated username is already present, the user has the chance of editing the generated user name since it is in the form.
